src/screens/Assets/wallet/WalletListItem.js

```diff
@@ -29,30 +29,42 @@ import FiatChangeView from 'components/modern/FiatChangeView';
 import TokenIcon from 'components/modern/TokenIcon';
 
 // Selectors
-import { useRates, useFiatCurrency } from 'selectors';
+import { useFiatCurrency, useChainRates } from 'selectors';
 
 // Utils
 import { getRate } from 'utils/assets';
 import { formatTokenValue, formatFiatValue } from 'utils/format';
 import { useThemeColors } from 'utils/themes';
 import { spacing } from 'utils/variables';
 
+// Types
+import type { Chain } from 'models/Chain';
+
 type Props = {|
   title: ?string,
   iconUrl: ?string,
   value: BigNumber,
   change?: BigNumber,
   symbol: string,
+  chain: Chain,
   onPress?: () => mixed,
 |};
 
-function WalletListItem({ title, iconUrl, value, change, symbol, onPress }: Props) {
+function WalletListItem({
+  title,
+  iconUrl,
+  value,
+  change,
+  symbol,
+  onPress,
+  chain,
+}: Props) {
   const colors = useThemeColors();
 
-  const rates = useRates();
+  const chainRates = useChainRates(chain);
```

Comment:
Consider creating simplier hook
```suggestion
  const rate = useTokenRate(chain, symbol); // expressed in user currency.
```

Reply:
Mate, tbh given the time we have I would consider this as future improvement as there are more important issues piling up 👀  hope it makes sense